src/Development/Shake/Demo.hs

```diff
@@ -63,7 +65,7 @@ demo auto = do
     createDirectoryIfMissing True dir
     forM_ ["Build.hs","main.c","constants.c","constants.h","build" <.> if isWindows then "bat" else "sh"] $ \file ->
         copyFile (manual </> file) (dir </> file)
-    when (not isWindows) $ do
+    unless isWindows $ do
          p <- getPermissions $ dir </> "build.sh"
```

Comment:
In general, I often find when reads better. Not a big deal here, but not a rule I like to apply universally.